src/CitizenProject/CitizenProjectCommentCommand.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace AppBundle\CitizenProject;
+
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\CitizenProject;
+use Symfony\Component\Validator\Constraints as Assert;
+
+class CitizenProjectCommentCommand
+{
+    /**
+     * @var CitizenProject
+     *
+     * @Assert\NotNull()
+     */
+    private $citizenProject;
+
+    /**
+     * @var Adherent Any member of the citizen project
+     *
+     * @Assert\NotNull()
+     */
+    private $author;
+
+    /**
+     * @var string|null
+     *
+     * @Assert\NotBlank()
+     */
+    private $content;
+
+    public function __construct(CitizenProject $citizenProject, Adherent $author)
```

Comment:
`?Adherent` because it might be null on editing.

Reply:
Editing is not something we want yet.
But anyway, the author is mandatory at object creation only, then it can be set as Anonymous and edited without calling the constructor (Thank you Doctrine !)

Reply:
This is not the entity but the command.

Reply:
Good point my bad !
Edit is not feature yet so we can let it go, no ?

Reply:
Please do it now, it's not that easy to catch this business constraint.